src/ui/setup/prefs.ts

```diff
@@ -43,20 +49,17 @@ export function updatePrefs(state: UIState, oldState: UIState) {
   updatePref("selectedPanel", getSelectedPanel);
   updateAsyncPref("eventListenerBreakpoints", (state: UIState) => state.eventListenerBreakpoints);
 
-  const previousReplaySessions = currentReplaySessions;
-  currentReplaySessions = updateReplaySessions(state, currentReplaySessions);
-
-  if (previousReplaySessions && previousReplaySessions !== currentReplaySessions) {
-    asyncStore.replaySessions = currentReplaySessions;
-  }
+  await updateReplaySessions(state);
```

Comment:
this might be a moderate perf slowdown. Kinda how @jcmorrow predicted :) 

I'm thinking it might be reasonable to remove the `await` here as long as we can guarantee ordering.

Reply:
Yeah there's nothing really waiting for this function so we can just axe it